beard/clustering/wrappers.py

```diff
@@ -80,8 +86,17 @@ def __init__(self, method="single", affinity="euclidean", threshold=None,
             See scipy.cluster.hierarchy.fcluster for further details.
 
         :param scoring: callable
-            The scoring function to maximize in (semi-)supervised clustering
-            (when y!=None).
+            The scoring function to maximize in either (semi-)supervised
+            clustering if y is not None, or unsupervised otherwise (using
+            "silhouette_score as an example when y is None or all y =-1).
+            In case of silhouette_score, the metric param of this function
+            must be set to 'precomputed'.
```

Comment:
If you want italics inside RTD, use stars instead of quotes:
http://docutils.sourceforge.net/docs/user/rst/quickref.html#inline-markup